_sass/vendor/_bootstrap.scss

```
@@ -4609,12 +4609,12 @@ tbody.collapse.in {
 
 .navbar-fixed-top .navbar-collapse,
 .navbar-fixed-bottom .navbar-collapse {
-  max-height: 340px;
+  min-height: 340px;
```

Actually, try only reverting this change since the media query below only applies to mobile.  That should fix it.  
```suggestion
  max-height: 340px;
```